### PR TITLE
PLATFORM-3401: fix test for GlobalTitle

### PR DESCRIPTION
**Pull request by harnash**
This should fix the tests and make it more reliable on DEV.

ping: @Wikia/core-platform-team, @mszabo-wikia

### CODE REVIEW

File: includes/wikia/GlobalFunctions.php [hunk 1]
```diff
@@ -1562,14 +1562,13 @@ function wfHttpsToHttp( $url ) {
...
-	if ( $parsedURL === false ) {
+	global $wgWikiaBaseDomain, $wgDevDomain, $wgWikiaEnvironment, $wgDevelEnvironment;
+	$host = parse_url( $url, PHP_URL_HOST );
+	if ( $host === false ) {
 		return false;
 	}
 
-	$host = $parsedURL['host'];
-	if ( !empty( $wgDevDomain ) ) {
+	if ( $wgDevelEnvironment && !empty( $wgDevDomain ) ) {
```
Comment (Grunny) [inline includes/wikia/GlobalFunctions.php:1571]: Do we need to check both?

**Review by Grunny — APPROVED** [review]
One question, otherwise looks good.

File: includes/wikia/GlobalFunctions.php (hunk 1 above)
Comment (harnash) [reply includes/wikia/GlobalFunctions.php:1571]: Probably don't but I'm not 100% sure that those two are set always

Comment (Grunny) [reply includes/wikia/GlobalFunctions.php:1571]: Both are set at the same time: https://github.com/Wikia/config/blob/dev/LocalSettings.php#L19-L21 but doesn't hurt I suppose. :)

Comment (harnash) [reply includes/wikia/GlobalFunctions.php:1571]: Especially on tests when they are being mocked out

**Pull request merged**